src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_TYPES = 5;
+    public static final int MAX_SIDES_LENGTH = 100;
+    public static final int DEFAULT_RADIUS = 10;
+    public static final Color DEFAULT_COLOR = Color.WHITE;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int choice = random.nextInt(FIGURE_TYPES);
+        int side1 = random.nextInt(MAX_SIDES_LENGTH) + 1;
+        int side2 = random.nextInt(MAX_SIDES_LENGTH) + 1;
+        int side3 = random.nextInt(MAX_SIDES_LENGTH) + 1;
```

Comment:
generate random side lengths just in place of usage
code is duplicating, so extract `random.nextInt(MAX_SIDES_LENGTH) + 1;` to the separate method
```suggestion
```